pkg/asset/installconfig/platform.go

```diff
@@ -18,6 +20,27 @@ const (
 
 var (
 	validPlatforms = []string{AWSPlatformType, LibvirtPlatformType}
+
+	validAWSRegions = map[string]string{
+		"ap-northeast-1": "Tokyo",
+		"ap-northeast-2": "Seoul",
+		"ap-northeast-3": "Osaka-Local",
+		"ap-south-1":     "Mumbai",
+		"ap-southeast-1": "Singapore",
+		"ap-southeast-2": "Sydney",
+		"ca-central-1":   "Central",
+		"cn-north-1":     "Beijing",
+		"cn-northwest-1": "Ningxia",
+		"eu-central-1":   "Frankfurt",
+		"eu-west-1":      "Ireland",
+		"eu-west-2":      "London",
+		"eu-west-3":      "Paris",
+		"sa-east-1":      "São Paulo",
+		"us-east-1":      "N. Virginia",
+		"us-east-2":      "Ohio",
+		"us-west-1":      "N. California",
+		"us-west-2":      "Oregon",
```

Comment:
This looks like something that could be a pain to keep up to date.  How about a TODO or something to consider getting the list from an AWS API?

Reply:
The installer doesn't require Internet access today. Reading this list from AWS would introduce that dependency.

Reply:
> How about a TODO or something to consider getting the list from an AWS API?

And also, AWS doesn't seem to provide an API for pulling the locations.  Previous discussion starting [here][1].

[1]: https://github.com/openshift/installer/pull/287#discussion_r219241607

Reply:
ok, thanks for the info and references